app/services/redis/redis_client.py

```
@@ -0,0 +1,243 @@
+        Args:
+            key: Redis key
+            value: Value to store
+            ex: Expiration time in seconds (optional)
+            nx: Only set if key doesn't exist (optional)
+
+        Returns:
+            True if successful, False otherwise
+        """
+        try:
+            return self._client.set(key, value, ex=ex, nx=nx)
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)